contributing_to_docs/doc_guidelines.adoc

```diff
@@ -162,7 +162,7 @@ When needed, use `.Prerequisites`, `.Next steps`, or `.Additional resources` syn
 
 When possible, generalize references to the product name and/or version using
 the `{product-title}` and/or `{product-version}` attributes. These attributes
-are pulled from distro mapping definitions in the
+are pulled by AsciiBinder from the distro mapping definitions in the
 https://github.com/openshift/openshift-docs/blob/master/_distro_map.yml[distro_map.yml]
```

Comment:
While you are here, it would be good to fix this filename:
s/[distro_map.yml]/[_distro_map.yml]

Reply:
Good catch! Thanks Bob. Resolved.